.github/workflows/ci.yml

```diff
@@ -260,6 +276,8 @@ jobs:
       - name: Composer
         env:
           INPUTS_COMPOSER_REQUIRE_EXTRA: ${{ inputs.composer_require_extra }}
+          INPUTS_PRESERVE_VENDOR_TEST: ${{ inputs.preserve_vendor_test }}
+          
```

Comment:
```suggestion
```
It's a bool so we don't need this as an environment variable.  The whole reason for putting inputs in as environment variables was to prevent string substitution attacks, you can't do that with bool

Reply:
Removed